### ISSUE TITLE
Ongoing assistance with improving quality of English translations

### ISSUE THREAD

**Issue by hansbkk** [cozy/cozy-home#286]
Hello, offered my services via email to Frank Rousseau, as a literate NES with translation experience to help make the Cozy interfaces read more naturally for the English-speaking market.

Have no idea about how to find the files containing the strings that need fixing as I come across them, looking for guidance on how to use this Git thing.

Notes from my initial signing in to the cloud service:

#1 "information" is not countable - can't be plural. Also not the right word - "Add more data vs Send just this data for now" would be more appropriate.

#2 "we recommend you to install"
  remove the "to" 

#3 "browser your files"
  just browse

**Comment by frankrousseau** [comment]
Hello Hans,

Thank you for your assistance and welcome to the Cozy project.

Here is my recommandation to work on fixing English wording in Cozy:

*Editing*

* Fork the repository by clicking on the button located at the top of the screen on the right.
* Once you have your copy of the repository, browse your files from the browser to reach the `client/app/locales/en.coffee` file.
* Click on the pen icon (just above the file content, on the right) to edit the file.
* Once your done with edition, click on the commit changes button located at the bottom of the page.

*Submitting your changes*

* Now your changes are saved, you can submit them by doing a *Pull Request*. It's an operation to ask to the main repository owner for merging your changes.
* To make a *Pull Request*, you have to click on the third icon on the right of the screen. 
* Then, click on the *New pull request button*. Enter details and explanation about what you propose. And you're done. 
* Wait for your *Pull Request* to be reviewed and merged.

It looks complicated at first sight, but every steps of the process make sense.

How does it sound to you?

**Comment by hansbkk** [comment]
I looked at the file before and don't recall seeing any of the strings above that need correcting. Are some strings stored in other locations?

Also, I will most likely only be willing to take the time on this if I become a regular user of the project, and my immediate need is for rock-solid CardDav services that allow for full implementation and fidelity to the "Google/Apple interpretation" of the vCard v3.0 standard. From my initial (pretty casual) evaluation it seems there are problems in that area that may well be deal-killers for me, at least for the moment.

**Comment by frankrousseau** [comment]
Ok to put the priority on CardDav problems, we will work on it as soon as possible. We'll keep you informed.

As a reminder, here are the other locale files:

* Login and register screens: https://github.com/cozy/cozy-proxy/blob/master/client/locales/en.coffee
* Contacts: https://github.com/cozy/cozy-contacts/blob/master/client/app/locales/en.coffee
* Calendars: https://github.com/cozy/cozy-calendar/blob/master/client/app/locales/en.coffee
* Files: https://github.com/cozy/cozy-files/blob/master/client/app/locales/en.coffee
* Emails: https://github.com/cozy/cozy-emails/blob/master/client/app/locales/en.coffee
* Photos: https://github.com/cozy/cozy-photos/blob/master/client/app/locales/en.coffee
* Sync: https://github.com/cozy/cozy-sync/blob/master/server/views/index_en.jade

**Comment by hansbkk** [comment]
"As a reminder" LOL I had no idea, thought there was just the one 8-)

I just posted a long rant/screed over there, do please just pick and choose whatever might be helpful and ignore the rest, I do tend to be too opinionated 8-)

Thanks so much for such a response, very encouraging.

I'll also be checking out owncloud, apparently their web-based editor has a lot of flexibility. They use SabreDAV "out of box" as their CardDAV server, and that does use a "folder of VCFs" as the storage back-end.

**Comment by frankrousseau** [comment]
Yes, Owncloud is written with PHP. We use Javascript/Node.js. We don't store Contacts as VCF. Cozy stores every data in a database to make it easier to use by another application.

**Comment by frankrousseau** [comment]
It provides URLs where all Cozy contacts are available in CarDAV format. Cozy is a technology based on a REST architecture, it doesn't rely much on files.

**Comment by hansbkk** [comment]
Well as I stated I don't know (nor care) much about underlying
code/language/architecture issues, I'm just a potential consumer / user of
your services, and letting you know from that POV:

I need to know none of the information stored in my contacts data set will
be lost by storing/sync'ing via Cozy. This is a "would be nice" feature I
am able to compromise on, if/when it becomes available then I will re-visit
the Cozy project.

I was making suggestions on how to accomplish that (I thought) important
goal more easily. I still don't see why the *sync process* can't supporting
"native VCF" as the canonical data store.

You can then do whatever you like on the database side, just support a
subset of fields for your web-based editor (which I couldn't use) and
easier integration with your other apps.

Going this way you just need to make sure when data is edited in your DB
subset, that gets exported out to the sync store (native vCard files) in
such a way that the unsupported data fields aren't disturbed.

Anyway, it's your project I'm just a noob prospect tossing out ideas here.

**Comment by hansbkk** [comment]
Sorry, correction in bold:

------------
Well as I stated I don't know (nor care) much about underlying
code/language/architecture issues, I'm just a potential consumer / user of
your services, and letting you know from that POV:

I need to know none of the information stored in my contacts data set will
be lost by storing/sync'ing via Cozy. This *is NOT a* "would be nice"
feature I am able to compromise on, if/when it becomes available then I
will re-visit the Cozy project.

**Comment by frankrousseau** [comment]
Hi Hans, 

We totally agree, that information should not be lost and Contacts should be compatible with as much as application possible. We'll work on debugging the issue you reported.

I understand you had a bad experience and it's very sad for both of us. Sad for the Cozy team because your  made very precise reports. That was really helpful. Sad for you because Cozy allows you to do many things while preserving your privacy. As an example, contacts can be reused when you want to share files or when you write an email with the Cozy webmail. 

I hope we will manage to fix the bugs you reported and will notify you once it's done.

**Comment by hansbkk** [comment]
Not at all "sad" Frank, I'm just testing different CardDAV implementation services with dummy data for now, and believe me this is par for the course in this part of the web standards woods, sync'ing contact info is inherently hard.

FFR to incoming googlers: more general issue here: cozy/cozy-sync#26
